menu-screen.lua

```diff
@@ -177,6 +173,26 @@ function MenuScreen:drawPlayerScreens()
 	end
 end
 
+function MenuScreen:drawShinyTitle()
+	self.font = love.graphics.newFont("font/Retro Computer_DEMO.ttf", 72)
```

Comment:
same thing here. Never call `new`-something besides in the `initialize()` function